src/app/rest/rest.service.ts

```diff
@@ -24,7 +24,8 @@ export class User {
 let SERVER_ADDR = location.hostname;
 
 export var HTTP_SERVER_ROOT;
-export var REST_SERVICE_ROOT;
+export var REST_SERVICE_V2_ROOT;
```

Comment:
Don't create two version for that. It increases the complexity. Change the backend side.